OWNERS_ALIASES

```diff
@@ -0,0 +1,35 @@
+aliases:
+    platform:
+    - zdtsw
+    - etirelli
+    - LaVLaS
+    - VaishnaviHire
+    - AjayJagan
+    - ykaliuta
+    - ajaypratap003
+    - mattmahoneyrh
+    - asanzgom
+    serving:
+    - VedantMahabaleshwarkar
+    - israel-hdez
+    - danielezonca
+    ide:
+    - harshad16
+    dashboard:
+    - andrewballantyne
+    - lucferbux
+    datasciencepipelines:
```

Comment:
@HumairAK should we have at least one more owner?

Reply:
yeah, add @gmfrasca